framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -132,6 +132,9 @@
     "assumeAssertionsAreEnabled",
     "assumeAssertionsAreDisabled",
 
+    // Whether to assume receiver and params of the method to be non-null after the method
```

Comment:
As this is only relevant for the Nullness Checker, move this there: https://github.com/eisop/checker-framework/blob/c62ed408a5a957b86457c776f4aeaf588f789f00/checker/src/main/java/org/checkerframework/checker/nullness/NullnessChecker.java#L38